example-app/app/build.gradle

```diff
@@ -44,17 +44,6 @@ android {
     }
 
     flavorDimensions "lang"
```

Comment:
I think `flavorDimensions` is not needed since `productFlavors` have been removed

Reply:
Done